modules/database/audit-autoloaded-options/update-autoload.js

```diff
@@ -0,0 +1,36 @@
+/* eslint camelcase: "off", no-alert: "off" */
+/* global jQuery:false, perflabAutoloadSettings:false */
+
+( function ( $, document ) {
+	$( document ).on( 'click', '.update-autoload', function ( e ) {
+		e.preventDefault();
+		const button = $( this );
+		const optionName = button.data( 'option-name' );
+		const autoload = button.data( 'autoload' );
+		const value = button.data( 'value' );
+		const data = {
+			option_name: optionName,
+			autoload,
+			value,
+		};
+
+		$.ajax( {
+			url:
+				perflabAutoloadSettings.root +
+				'perflab-aao/v1/update-autoload/' +
+				optionName,
```

Comment:
Per above, I don't think this needs to be part of the route. It's already part of the `data` being sent anyway.

Reply:
Nice catch.